readme.md

```diff
@@ -15,7 +15,7 @@ The following table shows the Keycloak versions used by the embedded Keycloak Se
 Embedded Keycloak Server | Keycloak
 ---|---
 1.x.y | 9.0.3
-2.x.y | 10.0.2
+2.x.y | 11.0.0
```

Comment:
We might want to make it v3 of the adapter in order to keep v2 of the adapter Keycloak v10 compatible.